core/tests/protractor_desktop/voiceoverUploadFeatures.js

```diff
@@ -162,4 +159,8 @@ describe('Voiceover upload features', function() {
        ' The uploaded file is 301.87 seconds long.'}]);
     await users.logout();
   });
+
+  afterAll(async function() {
+
+  });
```

Comment:
I think you can leave this out all together

Reply:
Done!